ci-operator/step-registry/openshift/e2e/openstack/fipless/openshift-e2e-openstack-fipless-workflow.yaml

```diff
@@ -0,0 +1,19 @@
+workflow:
+  as: openshift-e2e-openstack-fipless
+  steps:
+    pre:
+      - ref: openstack-provision-machinesubnet
```

Comment:
this step fails to find the CLUSTER_NAME:
```
/bin/bash: line 14: /tmp/secret/CLUSTER_NAME: No such file or directory
```

Reply:
CLUSTER_NAME is written to SHARED_DIR in ci-operator/step-registry/openstack/conf/resourcenames/openstack-conf-resourcenames-commands.sh.
We'll need to run the `openstack-provision-machinesubnet` and `openstack-provision-bastionproxy` steps as part of the `ipi-openstack-pre` chain.